lib/rails_cursor_pagination/paginator.rb

```diff
@@ -444,7 +438,18 @@ def filtered_and_sorted_relation
       memoize :filtered_and_sorted_relation do
         next sorted_relation if @cursor.blank?
 
-        sorted_relation.where "#{sql_column} #{filter_operator} ?", filter_value
+        unless custom_order_field?
+          next sorted_relation.where "#{id_column} #{filter_operator} ?",
+                                     decoded_cursor_id
+        end
+
+        sorted_relation
+          .where("#{@order_field} #{filter_operator} ?", decoded_cursor_field)
+          .or(
+            sorted_relation
+              .where("#{@order_field} = ?", decoded_cursor_field)
+              .where("#{id_column} #{filter_operator} ?", decoded_cursor_id)
+          )
```

Comment:
Since this is a very crucial part of the algorithm, what about adding some more description to the method comment? Also repeating the example you put in the readme may be enough.

Reply:
Sure, can do :)